Client/RecommendationRequestClient.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * This file is part of the EzSystemsRecommendationBundle package
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributd with this source code.
+ */
+namespace EzSystems\RecommendationBundle\Client;
+
+interface RecommendationRequestClient
```

Comment:
Please add a short meaningful description of what the interface is for. It doesn't have to be extensive, one or two lines will do.

I might not have done it in the other one, but I should have in that case ;-)